test/runtests.jl

```diff
@@ -28,9 +29,9 @@ Aqua.test_all(CompatHelper; ambiguities=false)
     @test occursin(Regex("\\sversion = \"$(major_version)\"\n"), workflow_filecontents)
     @test length(findall(r"version[\s]*?=", workflow_filecontents)) == 1
 end
-
 include("patches.jl")
 
+cwd = pwd()
```

Comment:
This isn't used
```suggestion
```

Reply:
it's used at the top of the `tests/utilities/new_versions.jl` file to avoid an error where we are cd'd into a tempdir that gets deleted. There is possibly a better solution, but would require a rework of many tests.

Reply:
```suggestion
const cwd = pwd()
```

Reply:
the const is fine, it needs to grab the pwd before the tests run.